gulp/tasks/copy.js

```diff
@@ -16,9 +16,7 @@ function _genericCopy( src, dest ) {
     .pipe( gulpChanged( dest ) )
     .on( 'error', handleErrors )
     .pipe( gulp.dest( dest ) )
-    .pipe( browserSync.reload( {
-      stream: true
-    } ) );
+    .pipe( runBrowserSyncApp() );
```

Comment:
I'm confused about why this is still needed here. Doesn't having browsersync running via gulp-watch nullify the need for this (and in the scripts and style tasks, too)?

Reply:
Yeah, I think you have it right Ans. I'll give this a test later today and report back.

Reply:
@jimmynotjim did you have a chance to verify this?

Reply:
Couldn't the `reload` call happen in `watch` as part of the actions it takes when it detects a change?

Reply:
I'm actually on the side of "lets remove browser sync entirely" now. When I was testing this I noticed that the entire build process would be kicked off and as it worked it's way through everything the page would refresh multiple times, making it take longer than simply running the appropriate task and refreshing manually. I think we ought to simplify the build process as much as possible, then look at adding features back in as needed.

Reply:
Okay, I'll update this to remove browserSync and heavily spam the FEWD channel with news of its impending merge to solicit any latent protest.